### ISSUE TITLE
Missing label for input

### ISSUE THREAD

**Issue by adr29truck** [adr29truck/APL-prep#5]
Looks like there's a label missing for this input. That makes it hard for people using screen readers or voice control to use the input.

_Originally posted by @accesslint[bot] in https://github.com/adr29truck/APL-prep/pull/4#discussion_r559974802_
